python/packages/nisar/workflows/static.py

```diff
@@ -0,0 +1,378 @@
+
+            # Get DEM and water mask descriptions, if found in the input raster
+            # datasets.
+            dem_description = get_raster_dataset_metadata_item(
+                dem_raster_file, "dem_description", default="(NOT SPECIFIED)"
+            )
+            water_mask_description = get_raster_dataset_metadata_item(
+                water_mask_raster_file,
+                "water_mask_description",
+                default="(NOT SPECIFIED)",
+            )
+
+            # Populate the 'grids' group.
+            logger.info("Populate raster layers and grid coordinates in output HDF5")
+            grids_group = instrument_group.create_group("STATIC/grids")
+            dataset_creation_kwds = build_hdf5_dataset_creation_kwds_dict(
+                dataset_shape=(geo_grid.length, geo_grid.width),
+                **output_params["dataset"]
+            )
+            with log_elapsed_time(logger.info, "Writing raster layers to output HDF5"):
+                populate_grids_group(
+                    grids_group=grids_group,
+                    product_spec=product_spec,
+                    dataset_creation_kwds=dataset_creation_kwds,
+                    reprojected_dem=reprojected_dem,
+                    layover_shadow_mask=layover_shadow_mask,
+                    local_incidence_angle=local_inc_angle,
+                    line_of_sight_x=los_east,
+                    line_of_sight_y=los_north,
+                    water_mask=binary_water_mask,
+                    rtc_gamma_to_sigma_factor=gamma0_to_sigma0_factor,
+                    rtc_gamma_to_beta_factor=gamma0_to_beta0_factor,
+                    geo_grid=geo_grid,
+                    dem_disclaimer=dem_description,
+                    water_mask_disclaimer=water_mask_description,
+                )
+
+            # Populate the 'metadata' group.
+            metadata_group = instrument_group.create_group("STATIC/metadata")
+            populate_metadata_group(
+                metadata_group=metadata_group,
+                product_spec=product_spec,
+                orbit=orbit,
+                attitude=attitude,
+                native_doppler=native_doppler,
+                radar_grid=radar_grid,
+                software_version=isce3.__version__,
+                dem_source=dem_description,
+                water_mask_source=water_mask_description,
+                runconfig_contents=params,
+            )
+
+    logger.info("Done")
+
+
+def main(args: Sequence[str] | None = None) -> None:
+    """
+    Main command line entrypoint.
+
+    Parameters
+    ----------
+    args : sequence of str or None, optional
+        The list of arguments. If None, the argument list is taken from `sys.argv`.
+        Defaults to None.
+    """
+    # Setup the argument parser.
+    parser = argparse.ArgumentParser(description="Run the NISAR Static Layers workflow")
+    parser.add_argument(
+        "config_file", type=Path, help="The run configuration YAML file"
```

Comment:
Could we be specific about the contents of the file?

```suggestion
        "config_file", type=Path, help="Run configuration YAML for the STATIC workflow"
```